modules/ossm-config-sidecar-mtls.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-config.adoc
+
+[id="ossm-security-mtls-sidecars-incoming-services_{context}"]
+=== Configuring sidecars for incoming connections for specific services
+
+You can also configure mTLS for individual services by creating a policy. 
+
+.Procedure
+
+. Create a YAML file using the following example. 
++
+.PeerAuthentication Policy example policy.yaml
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: default
+  namespace: <namespace>
+spec:
+  mtls:
+    mode: STRICT
+----
++
+.. Replace `<namespace>` with the namespace where the service is located. 
+
+. Run the following command to upload the resource to the namespace where the service is located. It must match the `namespace` field in the Policy resource you just created.
```

Comment:
Could be me, but "upload the resource to the namespace" sounds awkward.... is something like "create the resource in the namespace" better?